src/ngraph/builder/reshape.cpp

```diff
@@ -195,6 +195,29 @@ shared_ptr<Node> builder::opset1::reorder_axes(const Output<Node>& value, vector
         ->add_provenance_group_members_above({value});
 }
 
+shared_ptr<Node> builder::opset1::squeeze(const Output<Node>& value, vector<size_t> axes)
+{
+    if (axes.empty())
+    {
+        return value.get_node_shared_ptr();
+    }
+
+    Shape in_shape{value.get_shape()};
+    for (size_t idx = 0; idx < axes.size(); ++idx)
+    {
+        in_shape.at(axes.at(idx)) = 0;
+    }
+    Shape output_shape;
+    for (auto axis : in_shape)
```

Comment:
This loop can be replaced with `std::copy_if`

Reply:
I've overlooked that there indeed is one overload since c++11 :)